georasters/georasters.py

```diff
@@ -303,14 +290,6 @@ def __init__(self, raster, geot, nodata_value=np.nan, fill_value=-1e10, projecti
                                              fill_value=fill_value)
         self.geot = geot
         self.nodata_value = nodata_value
```

Comment:
Would there be a reason to not use the ``@property``  for the other properties?

Reply:
There's no need, unless you want to add e.g. type checks when these get set.
Main reason to make those other attributes into properties is, they are derived values and should not be stored by itself but rather looked up from their "parents".